site/docs/4.4/utilities/interactions.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+title: Interactions
+description: Utility classes that change how users interact with contents of a website.
+group: utilities
+---
+
+## Text selection
+
+Change the way in which the content is selected when the user interacts with it.
+
+{% capture example %}
+<p class="user-select-all">This paragraph will be entirely selected when clicked by the user.</p>
+<p class="user-select-auto">This paragraph has the default select behavior.</p>
+<p class="user-select-none">This paragraph will not be selectable when clicked by the user.</p>
+{% endcapture %}
+{% include example.html content=example %}
+
+Customize the available classes by changing the `$user-selects` Sass list in `_variables.scss`.
+
+## Cursors
+
+Change the cursor from a default value to `pointer` with our built-in attribute utility, `role="button"`. Add this attribute to an element to convey semantics to your visitors and add an explicit pointer cursor to help indicate elements are interactive.
+
+{% capture example %}
+<span role="button">Non-button element button</span>
+{% endcapture %}
+{% include example.html content=example %}
+
+Customize the available classes by changing the `$cursors` Sass list in `_variables.scss`.
```

Comment:
Since this part only mentions `[role="button"]`, I'd be in favor of dropping it totally.

Using this `role` has multiple prerequisite, like ensuring the interaction layer respects the `button` pattern: activation with <kbd>Space</kbd> and <kbd>Enter</kbd>, at least. This requires JavaScript and has very strong accessibility concerns: I wouldn't recommend to use this attribute to change the `cursor`.

Adding this `cursor` thing to our reboot seems good to prevent people from adding it manually… But this is more a bad practice than anything, and a `button` element should be used instead.We cannot recommend this.

BTW the last sentence is outdated anyway, isn't it?